propolis/src/dispatch/mod.rs

```diff
@@ -375,12 +405,20 @@ impl DispCtx {
         cb(&inst)
     }
 
+    /// Halts the instance asssociated with the dispatcher.
+    // TODO: What prevents re-invoking this method concurrently / repeatedly?
+    // Would that be safe? Kinda seems like invalid state transitions would
+    // cause a panic.
```

Comment:
Halt (and reset, for your subsequent comment) are still new additions and the interfaces are being polished